jobs/README.md

```diff
@@ -1,119 +0,0 @@
-# Datacube EKS Datacube Management Tasks
```

Comment:
This is a good doco should we move it elsewhere?

Reply:
That's fine most of them is covered on datakube-apps repo for database creation and indexing